statistics/update_test.go

```diff
@@ -148,18 +148,18 @@ func (s *testStatsUpdateSuite) TestMultiSession(c *C) {
 		testKit2.MustExec("delete from test.t1 limit 1")
 	}
 
-	err = testKit.Se.Close()
```

Comment:
Don't move this code. This is to test the session will update the stats even if it is closed.